docs/operations/storageclasses.md

```diff
@@ -122,3 +122,44 @@ parameters:
   csi.storage.k8s.io/secret-name: storageos-api
   csi.storage.k8s.io/secret-namespace: storageos
 ```
+
+### Topology volume Storage Class
+
+StorageClass that utilizes kubernetes node topology zones and a list of
```

Comment:
I would change this to something like:
> StorageClass that restricts volume provisioning to specific topologies, as described by Kubernetes here (insert link) 

As preferred and required zones can't actually be selected in k8s

Reply:
I like that 👌